Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -635,6 +636,15 @@ eval TerminateApp state = do
   pure $ terminateApp state.props.currentStage true
   continue state
 
+eval (CountDown seconds id status timerID) state = do 
+  if status == "EXPIRED" then do
```

Comment:
Can you confirm what is expected behaviour here.

Reply:
once the 5sec timer expires then we have to show this message